syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -202,11 +270,26 @@ def eq(self, _self, other):
     @staticmethod
     @overloaded.module
     def torch(module):
+        def add(self, other):
+            return self.__add__(other)
+
+        module.add = add
+
+        def sub(self, other):
+            return self.__sub__(other)
+
+        module.sub = sub
+
```

Comment:
Have you added tests for those "redirections"?

Reply:
I did